### ISSUE TITLE
Cannot integrate responsive table in the statement.

### ISSUE THREAD

**Issue by Butanium** [CodinGame/codingame-game-engine#42]
Hello I'm making a game and some information about bot class are shown in a table to the player in the statement.
Unfortunately, there is too many data in it :
![GIwVWHz61l](https://user-images.githubusercontent.com/55806347/137099762-2a528665-4950-4082-935d-d8e22f4fab90.gif)
Why is the auto overflow-x not allowed in statements ? It would be really useful.

**Comment by CGjupoulton** [comment]
Indeed. I have whitelisted everything the sanitzation library permits. I'm unsure why it doesn't allow overflow-x... the library isn't up to date on recent CSS properties either so the next step is probably to find a different library. Might take a while.

**Comment by Butanium** [comment]
Any news ?

**Comment by CGjupoulton** [comment]
Yes, I saw that it was stated in the CSS Schema, and I have allowed everything I possibly can without custom code. However, as you saw, it's still not allowing the overflow. 

I have updated the dependency in case this solves thie issue.
